src/js/apis/c2c/TagsService.js

```diff
@@ -0,0 +1,31 @@
+function TagsService(api) {
+  this.api = api;
+}
+
+TagsService.prototype.add = function(document) {
+  if (document.type !== 'r') {
+    // As for now, only routes may be tagged.
+    // TODO return an error message
+    return;
```

Comment:
Codacy doesn't like this because the returned value is not consistent with the other returned value.
Perhaps I could simply remove this test as the tags button is shown only for routes?

Reply:
what is returned in the route case?

(don't have time for a review tonight, but I may have a look tomorrow)

Reply:
you're returning a promise, so here you should return a `Promise.reject(new Error('not supported'))` or whichever you use for promises